code/game/objects/items/stacks/sheets/sheet_types.dm

```diff
@@ -102,6 +102,12 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
 	recipes = GLOB.metal_recipes
 	return ..()
 
+/obj/item/stack/sheet/metal/suicide_act(mob/living/carbon/user)
+	user.visible_message("<span class='suicide'>[user] begins whacking themselves over the head with \the [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	new /obj/structure/girder(loc)
+	user.gib()
```

Comment:
I thought making easy to obtain gib suicides was bad for some reason?

Reply:
outdated, i fucked up and pushed an older version of the branch